iib/web/api_v1.py

```diff
@@ -99,37 +125,48 @@ def _get_add_args(payload, request, overwrite_from_index, celery_queue):
     ]
 
 
-def _get_safe_args(args, payload):
+def _get_safe_args(
+    args: List[Any],
+    payload: PayloadTypesUnion,
+) -> List[Union[str, List[str], bool, Dict[str, str]]]:
     """
     Generate arguments that are safe to print to stdout or log.
 
-    :param list args: argument generated by _get_add_args or _get_rm_args
-    :param dict payload: Payload from the remove request
+    :param list args: arguments for each api, that are not safe
+    :param PayloadTypesUnion payload: Payload from the IIB request
+    :return: List with safe to print arguments
+    :rtype: list
     """
     safe_args = copy.copy(args)
 
-    if payload.get('cnr_token'):
-        safe_args[safe_args.index(payload['cnr_token'])] = '*****'
+    if payload.get("cnr_token"):
```

Comment:
From README: 
```
Development / Coding Standards
- Single quotes are used instead of double quotes
```

Reply:
Thanks, I did not notice that change (I was doing some changes to this function, but returned it back).